### PR TITLE
Add Spanish version

### PR DESCRIPTION
**Pull request by huangsuyu**

### CODE REVIEW

**Review by duc811997 — APPROVED** [review]

File: translations/Whitepaper_ Binance Smart Chain_spanish_translation_VIOLA.md [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+
+Ethereum y algunas otras redes tales como MATIC Bor, TOMOChain, GoChain, xDAI, usan Prueba de Autoridad (Proof-of-Authority - PoA) o sus variantes en diferentes escenarios, incluidas la testnet así como la mainnet. La PoA proporciona alguna defensa contra el ataque del 51%, con eficiencia mejorada y con tolerancia para un cierto nivel de actores bizantinos (maliciosos o hackeados). Funciona como una opción fundamental fácil de escoger.
+
+Mientras tanto, el protocolo PoA es el más criticado por no ser tan descentralizado como el de PoW, puesto que los validadores, es decir, los nodos que se turnan para producir bloques, tienen todos autoridad y son propensos a la corrupción y a los ataques contra la seguridad. Otras blockchains tales como EOS y Cosmos introducen ambas diferentes tipos de prueba de participación delegada (Deputy Proof of Stake - DPoS) para permitir votar a los tenedores de tokens y elegir el grupo validador. Esto incrementa la descentralización y favorece la gobernanza comunitaria.
```
Comment (3lectr0n) [inline translations/Whitepaper_ Binance Smart Chain_spanish_translation_VIOLA.md:50]: entre parentesis creo que seria (Delegated Proof of Stake - DPoS)

File: translations/Whitepaper_ Binance Smart Chain_spanish_translation_VIOLA.md [hunk 2]
```diff
@@ -0,0 +1,341 @@
...
+
+Aquí se propone que en BSC se combine DPoS y PoA para consenso, de manera que:
+
+1. Los bloques sean producidos por un grupo limitado de validadores.
+2. Los validadores se turnen para producir bloques a la manera PoA, similar a Clique, el motor de consenso de Ehereum.
```
Comment (3lectr0n) [inline translations/Whitepaper_ Binance Smart Chain_spanish_translation_VIOLA.md:55]: 2. Los validadores se turnen para producir bloques a la manera PoA, similar a Clique, el motor de consenso de Ethereum.

**Review by duc811997 — COMMENTED** [review]
[Account 0 (1).txt](https://github.com/binance-chain/whitepaper/files/5862661/Account.0.1.txt)

**Review by duc811997 — COMMENTED** [review]
[Account 0 (1).txt](https://github.com/binance-chain/whitepaper/files/5862671/Account.0.1.txt)

**Review by HaoyangLiu — APPROVED** [review]

**Review by toptenfact — APPROVED** [review]

**Review by hamzawajahat — COMMENTED** [review]
done

**Review by jonesk7734 — APPROVED** [review]
😀

**Pull request merged**